### PR TITLE
feature: QR wallets support

### PR DESCRIPTION
**Pull request by gbarkhatov**
- Adds support for QR wallets (`isQRWallet` boolean field)
- Shows a tooltip on them
- Provides an external link documentation https://i.imgur.com/s1Zf4mv.png

### CODE REVIEW

File: src/app/components/Modals/ConnectModal.tsx [hunk 1]
```diff
@@ -112,31 +114,51 @@ export const ConnectModal: React.FC<ConnectModalProps> = ({
...
+                            data-tooltip-content="QR codes used for connection/signing"
+                            data-tooltip-place="top"
+                          >
+                            <AiOutlineInfoCircle />
+                          </span>
+                          <Tooltip id={name} />
+                        </div>
+                      )}
+                    </div>
+                    {isQRWallet && (
```
Comment (vitsalis) [inline src/app/components/Modals/ConnectModal.tsx:148]: Why add this additional documentation field? Can't we do the same as with other wallets, i.e. the `href` above?

Comment (gbarkhatov) [reply src/app/components/Modals/ConnectModal.tsx:148]: Usual wallets
- Check if it's inside the `window.provider`
- Yes - allow button to be selectable
- No - point to the documentation

Integrated QR code wallets cannot be checked with `window.provider`
- So if `clicking` on this button will always lead to a link, how can the user select it.
- If not - allow to select without the docs - how can the user check the docs?

Comment (vitsalis) [reply src/app/components/Modals/ConnectModal.tsx:148]: Aha I see, so no need to point to documentation, the documentation pop-up is basically to allow users to install the wallet.

Comment (gbarkhatov) [reply src/app/components/Modals/ConnectModal.tsx]: >Aha I see, so no need to point to documentation, the documentation pop-up is basically to allow users to install the wallet.
Removed

**Review by vitsalis — APPROVED** [review]

**Pull request merged**